containers/landing/store.vue

```diff
@@ -8,10 +8,11 @@
     </div>
 
     <div class="cards">
-      <Card
-        v-for="(item, i) in items"
-        :key="i"
+      <StoreCard
+        v-for="item in items.slice(0, 3)"
```

Comment:
Don't slice here, assume that the `items` array will be 3 in length. The API will have a limiting parameter which will provide that